packages/bootstrap/lib/globals.module.scss

```diff
@@ -0,0 +1,13 @@
+/**
+ * IMPORTANT
+ * Before adding styles to this class, think _very carefully_ about
+ * possible side effects this may have on apps. These styles will
+ * be applied to all apps that consume Kaizen
+ */
+
+/** 
+ * Component-level overrides 
+ */
+[data-kaizen-component] {
```

Comment:
This feels very smelly to me, and I'm not sure where I am landing w.r.t convenience and flexibility. Another thing to note is this has the same level of specificity as class selectors, so to consistently override these defaults the components wrapper override will have to bump specificity.

The alternative here is to expose a sass mixin in either this package, or another new one, and have all components depend on and consume that mixin. Components will have more control if they need to opt out. 

WDYT?

Reply:
Yeah, I think that’s OK though. I wonder if we could choose a name that’s more indicative of its purpose so that components can _feel_ like they can reasonably opt-out. Maybe something like:

```
data-kaizen-component-defaults
```

Reply:
Hey @makenosound, 

If we're adding an attribute to tag components like this, I'd love to keep the door open for using defo to collect info on components as they're added/removed from the page so we can start to get some analytics on component usage.

Would a goal like that affect the attribute name we pick?

Reply:
A tiny bit. It would need to be a `data-` attribute but apart from that we’re all good. Probably makes sense to do something like `data-kz-*` to establish a Kaizen namespace essentially.